React/Modules/RCTAccessibilityManager.m

```diff
@@ -100,6 +105,20 @@ - (void)didReceiveNewVoiceOverStatus:(__unused NSNotification *)notification
   }
 }
 
+- (void)accessibilityAnnouncementDidFinish:(__unused NSNotification *)notification
+{
+  NSDictionary *userInfo = notification.userInfo;
+  // Response dictionary to populate the event with.
+  NSDictionary *response = @{@"announcement": userInfo[@"UIAccessibilityAnnouncementKeyStringValue"],
```

Comment:
Use the variable names for this instead of the string (so userInfo[UIAccessibilityAnnouncementKeyStringValue]